test/unit/spec/signaling/v2/twilioconnectiontransport.js

```diff
@@ -20,7 +20,7 @@ describe('TwilioConnectionTransport', () => {
     [              // bandwidthProfile
       [undefined],
       [{}, {}],
-      [{ video: {} }, {}],
+      [{ video: {} }, { video: {} }],
```

Comment:
looks like this is producing different output than last revision - is  this intentional ?  `{ video: {} }` => `{}` seems okay too.

Reply:
@makarandp0 I did this to maintain consistency between the options object and RSP.